### ISSUE TITLE
Preview notebook  no more work after version  v2.2.4 / similar with guidelines

### ISSUE THREAD

**Issue by eliselavy** [C2DH/journal-of-digital-history#282]
For DH Nord monday, i retrograd in v2.2.4
<img width="935" alt="Screenshot 2021-11-05 at 09 57 18" src="https://user-images.githubusercontent.com/35602279/140485237-07f471c7-9663-4214-93ca-a377277eafb4.png">

**Comment by eliselavy** [comment]
i just put version 2.2.6 again as :

**Comment by eliselavy** [comment]
@danieleguido  i retrograd version in production and in dev as it was for the production on 18 october , it means v2.2.4 as i need to have a proper environment for #DHNORD monday.

**Comment by eliselavy** [comment]
for the preview notebook , should have at least one reference in the bibliography otherwise

```
    at B (ArticleBibliography.js:16)
    at Qo (react-dom.production.min.js:153)
    at Os (react-dom.production.min.js:261)
    at gu (react-dom.production.min.js:246)
    at vu (react-dom.production.min.js:246)
    at su (react-dom.production.min.js:239)
    at react-dom.production.min.js:123
    at t.unstable_runWithPriority (scheduler.production.min.js:19)
    at Hi (react-dom.production.min.js:122)
    at qi (react-dom.production.min.js:123)
```

```
      <Row>
        <Col md={{ span: 7, offset: 2 }}>
          <div id="bibliography" className="anchor" />
          <h2 >{t('bibliography')}</h2>
          <div dangerouslySetInnerHTML={{
            __html: articleTree
              .formatBibliograhpy()
              .replace(/(https?:\/\/[0-9a-zA-Z-./_:?=]+)([^0-9a-zA-Z-./]+)/g, (m, link, r) => `<a href="${link}" target="_blank">${link}</a>${r}`)
          }}/>
        </Col>
```

**Comment by danieleguido** [comment]
fixed in #284, is in current "latest" http://localhost:3000/en/notebook-viewer/JTJGcHJveHktZ2l0aHVidXNlcmNvbnRlbnQlMkZpYm0tZXQlMkZqdXB5dGVyLXNhbXBsZXMlMkZtYXN0ZXIlMkZ1bml0ZWQtbmF0aW9ucyUyRnNlbmVnYWxfcG9wdWxhdGlvbl90cmVuZHMuaXB5bmI=

**Comment by danieleguido** [comment]
fixed in current dev
